test/run_all_tests.sh

```diff
@@ -114,14 +114,21 @@ else
   all_tests_result=999
 fi
 
-echo
 if [ 0 -eq ${all_tests_result} ]; then
   echo 'All tests have passed!'
-  gcovr --print-summary --sort-percentage --exclude-throw-branches --delete \
-    --object-directory . \
-    --root src \
-    --exclude .*_error.* \
-  && rm ./a.out *.gcno
+    gcovr --print-summary --sort-percentage --exclude-throw-branches --delete \
+      --object-directory . \
+      --root src \
+      --exclude .*_error.* \
+      --exclude test \
+      --coveralls coverage.json \
+      --txt && rm ./a.out *.gcno
```

Comment:
would prefer `--txt` to be on its own line
```suggestion
      --txt \
  && rm ./a.out *.gcno
```